Cargo.toml

```diff
@@ -31,6 +31,7 @@ nom = "5.1"
 crossbeam-channel = "0.4"
 parking_lot = "0.10.0"
 rand = "0.7.3"
+native-tls = "0.2.4"
```

Comment:
I went with native_tls because it allows us not to be responsible for shipping an up-to-date TLS library to our clients, as this relies on the underlying system library.

Reply:
I think that sounds reasonable.